virtual_rainforest/core/core_schema.json

```diff
@@ -188,10 +188,17 @@
                "type": "object",
                "properties": {
                   "soil_layers": {
-                     "description": "Number of soil layers to simulate",
-                     "type": "integer",
-                     "exclusiveMinimum": 0,
-                     "default": 2
+                     "description": "Depth and number of soil layers to simulate",
+                     "type": "array",
+                     "items": {
+                        "type": "number"
+                     },
+                     "minItems": 1,
+                     "uniqueItems": true,

```

Comment:
I think these should be in snake_case like your other key names.

Reply:
Those are JSONSchema keywords - can't do anything 😄